File: C - language/Q19.c
#include <stdio.h>

int main() {
    int arr[]={1,2,3,4,5};
    int len=sizeof(arr)/sizeof(arr[0]);
    int count;
    int i;
    for (i=0;i<len-1;i++){
        if(arr[i]>arr[i+1]){
            count++;
        }
    }
    if(count==0){
        printf("true");
    }else {
        printf("false");
    }

    return 0;
};

File: javascript/Q19.js
var arr =[1, 2, 3, 4, 5];
var count=0;

for (i=0;i<arr.length;i++){
    if (arr[i]>arr[i+1]){
        count=count+1
    }
}

if(count==0){
    console.log("Array is Sorted in Ascending order")
}else {
    console.log("Array is not in Ascending order")
}